CHANGELOG.md

```diff
@@ -36,6 +36,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   - Removed dataType/version from api granule schema
   - Added `@cumulus/api/endpoints/granules` unit to cover duration overwrite
     logic for PUT/PATCH endpoint.
+- **CUMULUS-3033**
+  - Added default timeout for `granuleEsQuery` search operation to be 60 seconds.
+  - Added functionality to allow search operation timeout to be configurable.
+
+### Fixed
+
+- **CUMULUS-3033**
+  - Fixed `granuleEsQuery` to properly terminate if `body.hit.total.value` is 0.
```

Comment:
Is timeout still an issue after you make this change?  If not, we should remove the `timeout` update.

Reply:
https://github.com/nasa/cumulus/pull/3201/commits/c74d76ab0f8d7a18f77c21f1dc63954fdada4258